cumulus/polkadot-parachain/src/command.rs

```diff
@@ -317,17 +345,25 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 fn extract_parachain_id(id: &str) -> (&str, &str, Option<ParaId>) {
 	const KUSAMA_TEST_PARA_PREFIX: &str = "penpal-kusama-";
 	const POLKADOT_TEST_PARA_PREFIX: &str = "penpal-polkadot-";
+	const WESTEND_TEST_PARA_PREFIX: &str = "penpal-westend-";
```

Comment:
Westend Penpal (or Trappist) may be worth developing, but why is it needed for Collectives and Glutton?